### PR TITLE
New extension: `fantasy-premier-league-rankings` extension

### PR DESCRIPTION
**Pull request by hwb15**
## Description

This is an extension which allows a user to input their fantasy premier league team id and keep track of their rankings across all the leagues they are entered in.

## Screencast

https://user-images.githubusercontent.com/47414652/189547553-1dc0f772-aa34-4dbf-896e-c06f57172108.mp4

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/fantasy-premier-league-rankings/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  "icon": "icon.png",
+  "author": "hwb",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "rankings",
+      "title": "View My FPL Rankings",
```
Comment (pernielsentikaer) [inline extensions/fantasy-premier-league-rankings/package.json:15]:
```suggestion
      "title": "View My Rankings",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* The extension doesn't seem to load correctly; it's ended with `isLoading` always being true

fantasy-premier-league-rankings 2023-01-11 at 15 24 24

* You could consider adding a nice [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview)

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.



I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Awesome, thanks for the corrections 🙏

**Comment by hwb15** [comment]
> 

Hi @pernielsentikaer -Thank you! I still have a few things I want to change up before looking it's ready for a full review again 🙏

**Comment by hwb15** [comment]
> Just request when you're ready :)

👋 @pernielsentikaer, latest changes reflect on initial review w/ a empty view and I've added keywords to make sure the search functionality works as expected. I made substantial improvements on the api which has sorted the isLoading issues.

I believe it is ready for a second review 🙏

**Comment by pernielsentikaer** [comment]
Hi @hwb15 👋 

The initial load doesn't have an empty view; it stalls in a loading state.

https://user-images.githubusercontent.com/38033107/212913433-20a22d42-fa6f-4da0-a3ad-67c11d98456e.mp4

You can empty the cache here

Raycast-xcP2v7fp@2x

**Comment by hwb15** [comment]
> Hi @hwb15 👋
> 
> The initial load doesn't have an empty view; it stalls in a loading state.
> 
>  Raycast-SSf6qTWU.mp4 
> You can empty the cache here
> 
> Raycast-xcP2v7fp@2x

Hi @pernielsentikaer,

I've fixed it after removing cache and also moving hosting of my api which seemed to be causing most of the issues.

Thank you 🙏

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

In the next version, we can play with [useFetch](https://developers.raycast.com/utilities/react-hooks/usefetch#signature)s cache to maybe speed it even more up 😊

**Comment by hwb15** [comment]
> Hi 👋
> 
> Looks good to me, approved 🔥
> 
> In the next version, we can play with [useFetch](https://developers.raycast.com/utilities/react-hooks/usefetch#signature)s cache to maybe speed it even more up 😊

🎉!

Thanks for all your help @pernielsentikaer - I'm going to take a look over the useFetch caching and rid of axios for the next version 😁

**Pull request merged**